launch_frontend/launch_frontend/expose.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module which adds methods for exposing parsing methods."""
+
+import inspect
+from typing import Text
+
+substitution_parse_methods = dict({})
+action_parse_methods = dict({})
+
+
+def __expose_impl(name: Text, parse_methods_map: dict, exposed_type: Text):
+    """Implementats expose decorators."""
+    def expose_substitution_decorator(exposed):
```

Comment:
@ivanpauno nit: this isn't always decorating a substitution

Reply:
Already changed in #3